documentation/product/connectivity/topics/creating-new-connections.md

```diff
@@ -66,75 +66,76 @@ The Asset Selector menu has a built-in search feature allowing you to quickly fi
 
 10.	Select the Termination A asset from the Asset Picker menu.
 
-```{image} /product/connectivity/media/creating-new-connections/image10.jpg
+```{image} /product/connectivity/media/creating-new-connections/image10.jpeg
 :class: border-black
 ```
 
 11.	Click "Termination A Port" to open the Port Picker menu.
 
-```{image} /product/connectivity/media/creating-new-connections/image11.jpg
+```{image} /product/connectivity/media/creating-new-connections/image11.jpeg
 :class: border-black
 ```
 
 12.	Select the corresponding port from the selected asset to associate to the Connection.
 
-```{image} /product/connectivity/media/creating-new-connections/image12.jpg
+```{image} /product/connectivity/media/creating-new-connections/image12.jpeg
 :class: border-black
 ```
 
-:::{note}The ports populated in the Port Picker menu are populated based on the available ports on the asset in Hyperview. If you do not see expected available port in this menu, then you may need to configure the ports from the asset's Information > Network Ports page.
+:::{note}
+The ports populated in the Port Picker menu are populated based on the available ports on the asset in Hyperview. If you do not see expected available port in this menu, then you may need to configure the ports from the asset's Information > Network Ports page.
```

Comment:
"If you do not see expected available port..."
should be:
"If you do not see the expected available port..."
or:
"If you do not see the expected available ports..."

Reply:
Done, thank you!